modules/system/lang/en/lang.php

```diff
@@ -61,9 +63,11 @@
             'users' => 'Users',
             'system' => 'System',
             'social' => 'Social',
+            'backend' => 'Backend',
```

Comment:
I took it from branch 'l55upgrade'. My mistake. I wanted to use the most current version of language file